1008-datagrid-design-guidance.md

```diff
@@ -0,0 +1,34 @@
+---
+category: component design guidance
+state: approved
+created: 2024-10-22
```

Comment:
```suggestion
created: 2024-12-03
```

Reply:
The updated date has been changed to `2024-12-03`, do we need to update the created date too?

Reply:
I think both dates should be the date we publish.